### PR TITLE
Show current branch and link to tree on local/staging within site notice

### PR DESCRIPTION
**Pull request by MusikAnimal**
This makes it more clear whose work you're testing.

### CODE REVIEW

File: src/AppBundle/Twig/AppExtension.php [hunk 1]
```diff
@@ -85,11 +86,20 @@ public function gitShortHash(): string
...
         return exec('git rev-parse HEAD');
     }
+
+    /**
+     * Get the currently checked-out Git branch.
+     * @return string
+     */
+    public function gitBranch(): string
+    {
+        return exec('git rev-parse --symbolic-full-name --abbrev-ref HEAD');
```
Comment (samwilson) [inline src/AppBundle/Twig/AppExtension.php]: I think this should use [Process](https://symfony.com/doc/current/components/process.html) instead.

Comment (MusikAnimal) [reply src/AppBundle/Twig/AppExtension.php]: Ah man, if we were on Symfony 4.2 we could use `Process::fromShellCommandline('...')->run()`. Instead we need to instantiate first, two liner versus one. Unless there's an easier way?

Comment (MusikAnimal) [reply src/AppBundle/Twig/AppExtension.php]: 3-liner even, but actually I think with Symfony 4.2 that would only bring it down to 2 lines (after `run()` you still need to call `->getOutput()`). No biggie

**Comment by MusikAnimal** [comment]
@samwilson Changed it to use `Process`. About the bundle... would it include the siteNotice code too? Or would we just want to make a Twig helper to generate the "Git branch: master (sha)" message itself? I think that would be pretty neat!

**Comment by MusikAnimal** [comment]
This looks super easy to add to the bundle, doing that now!

**Comment by MusikAnimal** [comment]
This is turning into a rabbit hole. Better we at least wait until Event Metrics is on Symfony 4.2 so that we can use `Process::fromShellCommandline`. Meanwhile I feel a slight sense of urgency to get this deployed because there are so many different things being tested :/

**Comment by aezell** [comment]
I agree with @MusikAnimal in that we can pull this into the bundle at a later date.

**Comment by MusikAnimal** [comment]
The one code review issue was addressed and this is a low-risk, staging-only patch, so I'm going to boldly merge. Shame me accordingly :)

**Comment by MusikAnimal** [comment]
The one code review issue has been addressed, and this is a low-risk, staging-only patch, so I am going to boldly merge. Shame me accordingly :)

**Pull request merged**